package.json

```diff
@@ -22,10 +22,4 @@
     "url": "https://github.com/giliyoffe/Fibbage-Diconium/issues"
   },
   "homepage": "https://github.com/giliyoffe/Fibbage-Diconium#readme",
-  "dependencies": {
-    "express": "^4.17.1"
```

Comment:
i added these here because the deploy wasn't working when they weren't here.. are we sure it will work if they are removed?

Reply:
I'm not sure. But that's odd, if it won't work without unnecessary dependency.